### PR TITLE
add Uniarts Chain types

### PR DESCRIPTION
**Pull request by tianxiemaochiyu**

### CODE REVIEW

File: packages/apps-config/src/endpoints/testing.ts [hunk 1]
```diff
@@ -204,6 +204,13 @@ export function createTesting (t: TFunction): LinkOption[] {
       providers: {
         ZERO: 'wss://alphaville-0.zero.io'
       }
+    },
+    {
+      info: 'uniarts',
+      text: t('rpc.uniarts', 'Uniarts', { ns: 'apps-config' }),
+      providers: {
+        ZERO: 'wss://testnet.uniarts.me'
```
Comment (jacogr) [inline packages/apps-config/src/endpoints/testing.ts:212]: UniArts: as the key?

**Review by jacogr — COMMENTED** [review]
Thank you, one small comment.

File: packages/apps-config/src/endpoints/testing.ts (hunk 1 above)
Comment (jacogr) [inline packages/apps-config/src/endpoints/testing.ts:212]:
```suggestion
        Uniarts: 'wss://testnet.uniarts.me'
```

File: packages/apps-config/src/api/spec/uniarts.ts [hunk 2]
```diff
@@ -0,0 +1,214 @@
...
+  WorkId: 'u32',
+  Id: 'u32',
+  Name: 'Vec<u8>',
+  Value: 'Vec<u8>',
+  ReasonIndex: 'u32',
+  LottoIndex: 'u32',
+  LotteryKind: {
+    _enum: [
+      'Routine',
+      'TreasuryFunded(ReasonIndex)'
```
Comment (jacogr) [inline packages/apps-config/src/api/spec/uniarts.ts:21]:
```suggestion
      Routine: 'Null',
      TreasuryFunded: 'ReasonIndex'
```

File: packages/apps-config/src/api/spec/uniarts.ts [hunk 3]
```diff
@@ -0,0 +1,214 @@
...
+    ]
+  },
+  LottoResult: {
+    _enum: [
+      'Routine(AccountId, Balance)',
+      'TreasuryFunded(Balance)'
```
Comment (jacogr) [inline packages/apps-config/src/api/spec/uniarts.ts:27]:
```suggestion
      Routine: '(AccountId, Balance)',
      TreasuryFunded: 'Balance'
```

File: packages/apps-config/src/api/spec/uniarts.ts [hunk 4]
```diff
@@ -0,0 +1,214 @@
...
+    ]
+  },
+  Lottery: {
+    round: 'LottoIndex',
+    kind: 'LotteryKind',
+    jackpot: 'Balance',
+    next_ticket_id: 'u32',
+    players: 'BTreeMap<AccountId, bool>',
+    tickets: 'BTreeMap<u32, AccountId>',
+    result: 'Option<LottoResult<AccountId, Balance>>'
```
Comment (jacogr) [inline packages/apps-config/src/api/spec/uniarts.ts:37]:
```suggestion
    result: 'Option<LottoResult>'
```

File: packages/apps-config/src/api/spec/uniarts.ts [hunk 5]
```diff
@@ -0,0 +1,214 @@
...
+    memo: 'Vec<u8>',
+    expiration: 'Option<BlockNumber>'
+  },
+  Ownership: {
+    owner: 'AccountId',
+    fraction: 'u128'
+  },
+  ReFungibleItemType: {
+    Collection: 'u64',
+    Owner: 'Vec<Ownership<AccountId>>',
```
Comment (jacogr) [inline packages/apps-config/src/api/spec/uniarts.ts:167]:
```suggestion
    Owner: 'Vec<Ownership>',
```

**Comment by tianxiemaochiyu** [comment]
Thanks for the suggestion, I have updated the code and submitted.

**Review by jacogr — APPROVED** [review]
Thank you!

**Pull request merged**